src/corelibs/U2View/src/ov_msa/PhyTrees/MSAEditorTreeViewer.cpp

```diff
@@ -50,6 +50,12 @@ MSAEditorTreeViewer::MSAEditorTreeViewer(const QString &viewName, GObject *obj,
       msaTreeViewerUi(nullptr) {
 }
 
+MSAEditorTreeViewer::~MSAEditorTreeViewer() {
+    if (editor != nullptr && isSyncModeEnabled()) {
```

Comment:
`editor != nullptr` why do you need this check? I can't understand, why the `editor` in destructor could be removed or not removed yet?

Reply:
Sorry didn't get the question. Anyway this code is removed.

Reply:
Why do you need this check? I mean, `editor != nullptr`. In what case `editor` equals `nullptr`?

Reply:
There are 2 cases when editor may be null:

1) When tree view is created but editor is not yet assigned. We have a TODO for it in the code to avoid this state.

2) When MsaEditor is destroyed. The tree-view is be destroyed after the editor instance. 
This order looks wrong to me: the tree view must be destroyed after the MsaEditor, but the release sprint is not the best time to make such big refactoring.